### PR TITLE
Fixes Subscription service provider 

### PR DESCRIPTION
**Pull request by sven1103**
After the removal of Java modules support, the Subscription provider implementation broke. 

This enables the findability of the service via the legacy `META-INF/services` approach to expose service providers.

### CODE REVIEW

**Review by KochTobi — COMMENTED** [review]
I do not understand the problem this PR fixes. Could you elaborate?

File: vaadinfrontend/src/main/java/life/qbic/datamanager/Application.java [hunk 1]
```diff
@@ -82,6 +82,7 @@ public static void main(String[] args) {
     messageBus.subscribe(whenPasswordResetRequestSendEmail(appContext), PASSWORD_RESET);
 
     setupUseCases(appContext);
+    logger.error("Oh oh");
```
Comment (Steffengreiner) [inline vaadinfrontend/src/main/java/life/qbic/datamanager/Application.java:85]: I think this is a remnant from testing? :) 
```suggestion
```

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Nicely done! The PW Reset and account creation email sending works as intended. 
I would suggest removing the testing logging remnant and merging development into the branch then this can be merged 👍

**Review by Steffengreiner — APPROVED** [review]
Ship it! 🚢

**Pull request merged**